bedrock/pocket/templates/pocket/base.html

```diff
@@ -41,12 +41,22 @@
     <title>Pocket: {% block page_title %}{% endblock page_title %}</title>
   </head>
   <body>
+    {% block header %}
+      {% include 'pocket/includes/nav.html' %}
+    {% endblock %}
+
     <main class="main-container">{% block content %}{% endblock %}</main>
 
+    {% block footer %}
+      {% include 'pocket/includes/footer.html' %}
+    {% endblock %}
+
      {% block site_js %}
       {{ js_bundle('pocket-site') }}
      {% endblock %}
 
-     {% block js %}{% endblock %}
+     {% block js %}
+      {{ js_bundle('pocket-nav') }}
+     {% endblock %}
```

Comment:
**Question**: If this .js file is only handling events in the nav, should this block be moved to the nav itself? Not sure what pattern is best for this situation

Reply:
Scripts should typically be inserted in the `js` block at the end of each page, else they can block rendering